apps/molgenis-components/src/components/forms/InputRefList.vue

```diff
@@ -46,7 +46,7 @@
             :class="{ 'is-invalid': errorMessage }"
           />
           <label class="form-check-label" :for="`${id}-${row.name}`">
-            {{ flattenObject(getPrimaryKey(row, tableMetaData)) }}
+            {{ applyJsTemplate(row, refLabel) }}
```

Comment:
refLabel prop can be null or undefined

Reply:
no we don't; https://github.com/molgenis/molgenis-emx2/search?q=InputRefList or am i missing something here ?